src/app/settings/settings.component.ts

```diff
@@ -31,6 +34,9 @@ export class SettingsComponent extends RouterEnter {
   helpline = 'help@practera.com';
 
   termsUrl = 'https://images.practera.com/terms_and_conditions/practera_terms_conditions.pdf';
+  // controll profile image updating
+  imageUpdating = false;
+  acceptFileTypes = ['.jpeg', '.jpg', '.png'];
```

Comment:
Don't use these file types, use `this.filestackService.getFileTypes(type)` instead